airflow/dag_processing/processor.py

```diff
@@ -559,6 +560,28 @@ def update_import_errors(session: Session, dagbag: DagBag) -> None:
 
         session.commit()
 
+    @staticmethod
+    def update_configuration_errors(session: Session, dagbag: DagBag) -> None:
```

Comment:
Hmm naming is hard, "configuration" here led me to think this is something around configs in airflow.cfg especially the name of Model in `models/dagconfigurationerror.py`

- DagParsingErrors 
- AirflowInputError

or we just have a `AirflowUserError` and we can remove `import_error` table and use this instead where `ImportError` type is replaced a ParsingError?

(Just thinking it out loud -- no strong opinion on any of the above)

Reply:
I ❤️ the idea of more helpful errors though so happy to see this